dotcom-rendering/index.d.ts

```diff
@@ -744,44 +744,6 @@ interface ConfigType extends CommercialConfigType {
 	isPreview?: boolean;
 }
 
-interface ConfigTypeBrowser {
```

Comment:
This is a drive by change. Turns out this type is dead. It most likely died during the `App.tsx` wars.